pkg/config/config.go

```diff
@@ -531,7 +531,17 @@ func load(config Config, origin string, loadSecret bool) error {
 	loadedKeys := config.AllKeys()
 	for _, key := range loadedKeys {
 		if _, found := knownKeys[key]; !found {
-			log.Warnf("Unknown key in config file: %v", key)
+			//Check if any subkey with a .* is marked as known
```

Comment:
Maybe we can describe what happens more clearly here:
```go
// allow wildcard rules, e.g.: apm_config.* would allow all sub-keys of apm_config
```

Reply:
Done.